sdk/graphql-transport/src/mappers/transaction-block.ts

```diff
@@ -181,7 +184,7 @@ export function mapTransactionBlockToInput(
 	}
 
 	return {
-		txSignatures: data.txSignatures,
+		txSignatures: sigs,
```

Comment:
not sure if this is a vec<signature> or a string array.

Reply:
and the passed sigs are already base64 encoded, so I thought this should be good to go.